src/components/patients/searchTool/index.tsx

```diff
@@ -2,22 +2,25 @@ import React from "react";
 import { Icon, H3, Button } from "@blueprintjs/core";
 import "./style.css";
 import SearchItem from "components/patients/searchTool/searchItem";
+import SearchName from "components/patients/searchTool/searchName";
 
 interface searchForm {
   label: string;
   symbol: string;
   text: string;
 }
 
-const SearchTool = () => {
-  const newSearchForm: searchForm = {
-    label: "",
-    symbol: "",
-    text: ""
-  };
-  const [searchForms, setSearchForms] = React.useState([
-    newSearchForm
-  ] as searchForm[]);
+interface Props {
+  onSearch: Function;
+}
+
+const SearchTool = ({ onSearch }: Props) => {
+  const [searchForms, setSearchForms] = React.useState([] as searchForm[]);
+  const [advancedSearchStyle, setAdvancedSearchStyle] = React.useState(
+    "hidden"
+  );
+
+  let [nameSearch] = React.useState({ text: "" });
```

Comment:
Why do you have a state without a setter?
Why is there an object with only a string field into it?

Reply:
corrected